TTS/tts/layers/xtts/trainer/gpt_trainer.py

```diff
@@ -237,8 +237,7 @@ def test_run(self, assets) -> Tuple[Dict, Dict]:  # pylint: disable=W0613
                     self.config,
                     s_info["speaker_wav"],
                     s_info["language"],
-                    gpt_cond_len=3,
-                    decoder="ne_hifigan",
+                    gpt_cond_len=3
```

Comment:
Trailing comma according to code style:
```suggestion
                    gpt_cond_len=3,
```